.demo/env/charts/gatekeeper.yaml

```diff
@@ -0,0 +1,75 @@
+charts:
+    gatekeeper:
+        enabled: false
+        constraints:
+            - parameters:
+                enabled: false
+                cpu: 10m
+                memory: 120Mi
+              policyName: containerlimits
+            - parameters:
+                enabled: true
+                tags:
+                    - ''
+                    # - 0.3.7-debian-10-r103
+                    # - 5.0.7-debian-10-r32
+                    # - v1.1.0
+                    # - latest
+                    # - master
+              policyName: bannedimagetags
+            - parameters:
+                enabled: true
+                repos:
+                    - otomi
+                    - mcr.microsoft.com
+                    - k8s.gcr
+                    - eu.gcr.io
+                    - quay.io
+                    - gcr.io
+                    - ghcr.io
+                    - us.gcr.io
+                    - busybox
+                    - docker.io
+                    - goharbor
+                    - grafana
+                    - istio
+                    - drone
+                    - prom
+                    - bats
+                    - kiwigrid
+                    - dduportal
+                    - weaveworks
+                    - curlimages
+                    - kennethreitz
+                    - angelbarrera92
+              policyName: allowedrepos
+            - parameters:
+                enabled: false
+                runAsUser:
+                    # rule: 'RunAsAny'
+                    rule: 'MustRunAs'
+                    # rule: 'MustRunAsNonRoot'
+                    ranges: # testing
+                        - min: 1001
+                          max: 1001
+                        - min: 2
+                          max: 998
+              policyName: pspallowedusers
+            - parameters:
+                enabled: false
+                allowedHostPaths:
+                    # - pathPrefix: /var/
+                    #   readOnly: true
+                    - pathPrefix: /tmp/
+                      readOnly: false
+              policyName: psphostfilesystem
+            - parameters:
```

Comment:
antipattern policy...I already mentioned that before and that we will NOT be using it

Reply:
The policies we have right now are not final. We have to decide which policies we want to use. 
All the .rego files we have in the system are just used here for aiding in the design of the feature and definitely need to be revisited.